source/guide_jupyter-notebook.rst

```diff
@@ -0,0 +1,174 @@
+  Running setup.py install for pandocfilters ... done
+ Successfully installed MarkupSafe-1.1.1 Send2Trash-1.5.0 attrs-19.3.0 backcall-0.2.0 bleach-3.1.5 decorator-4.4.2 defusedxml-0.6.0 entrypoints-0.3 importlib-metadata-1.7.0 ipykernel-5.3.1 ipython-7.16.1 ipython-genutils-0.2.0 ipywidgets-7.5.1 jedi-0.17.1 jinja2-2.11.2 jsonschema-3.2.0 jupyter-1.0.0 jupyter-client-6.1.5 jupyter-console-6.1.0 jupyter-core-4.6.3 mistune-0.8.4 nbconvert-5.6.1 nbformat-5.0.7 notebook-6.0.3 packaging-20.4 pandocfilters-1.4.2 parso-0.7.0 pexpect-4.8.0 pickleshare-0.7.5 prometheus-client-0.8.0 prompt-toolkit-3.0.5 ptyprocess-0.6.0 pygments-2.6.1 pyparsing-2.4.7 pyrsistent-0.16.0 python-dateutil-2.8.1 pyzmq-19.0.1 qtconsole-4.7.5 qtpy-1.9.0 six-1.15.0 terminado-0.8.3 testpath-0.4.4 tornado-6.0.4 traitlets-4.3.3 wcwidth-0.2.5 webencodings-0.5.1 widgetsnbextension-3.5.1 zipp-3.1.0
+ [isabell@stardust ~]$
+
+.. note:: With the following command you can start the jupyter server to test the basic installation without having a config file.
+  ``jupyter notebook --no-browser --ip 0.0.0.0 --port 8888``
+  Without having set a password the server would generate a token instead which you use as a parameter in the url.
+  The token would then be displayed in the console output. You can also see if the selected port (in this case 8888) is available. Use another port for your configuration if this is the case. Terminate it with pressing ``Ctrl + c`` two times.
+
+Configuration
+=============
+
+Let's generate a config file.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ jupyter notebook --generate-config
+ Writing default config to: /home/isabell/.jupyter/jupyter_notebook_config.py
+ [isabell@stardust ~]$
+
+Modify the ``~/.jupyter/jupyter_notebook_config.py`` and add the following parameters.
+
+.. code-block:: ini
+ :emphasize-lines: 4,5
+
+ c.NotebookApp.allow_password_change = False
+ c.NotebookApp.ip = '0.0.0.0'
+ c.NotebookApp.open_browser = False
+ c.NotebookApp.port = 8888
+ c.ContentsManager.root_dir = '/home/isabell/'
+
+.. note:: The parameter ``c.ContentsManager.root_dir`` sets the root of the Jupyter file manger. So you could browse your hole user directory with this setting.
+  You may want to create a subfolder and then set the ``root_dir`` to ``/home/isabell/subfolder/``
+
+Set a password for a secure web access.
+
+.. code-block:: console
+ :emphasize-lines: 2,3
+
+ [isabell@stardust ~]$ jupyter notebook password
+ Enter password:
+ Verify password:
+ [NotebookPasswordApp] Wrote hashed password to /home/isabell/.jupyter/jupyter_notebook_config.json
+ [isabell@stardust ~]$
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/jupyter.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:jupyter]
+ command=jupyter notebook
+ autostart=true
+ autorestart=true
+
+After creating the configuration, tell :manual:`supervisord <daemons-supervisord>` to refresh its configuration and start the service:
```

Comment:
You should probably use the `supervisord.rst` snippet here.

Reply:
I didn't know this is possible. I also do this for the web backend.